controls/os_spec.rb

```diff
@@ -210,7 +210,11 @@
     its(:content) { should match 'install hfsplus /bin/true' }
     its(:content) { should match 'install squashfs /bin/true' }
     its(:content) { should match 'install udf /bin/true' }
-    its(:content) { should match 'install vfat /bin/true' }
+    # if efi is active, do not disable vfat. otherwise the system
+    # won't boot anymore
+    unless Dir.exist?('/sys/firmware/efi')
```

Comment:
@rndmh3ro maybe we have a bug here. `Dir.exist?` gets executed in the local context and not on the remote system. We should do this check via inspec resources

Reply:
Uh okay, I did not know this... I'll have to take a look.